### ISSUE TITLE
Column prefix seems not working

### ISSUE THREAD

**Issue by tontonskie** [typeorm/typeorm#757]
```
export class AppointmentReview {

  @Column({ name: '_rate' })
  rate: number;

  @Column('text', { name: '_text', nullable: true })
  text: string | null | undefined;

  @CreateDateColumn({ name: '_created_at' })
  createdAt: Date;

  @UpdateDateColumn({ name: '_updated_at' })
  updatedAt: Date;
}

export enum AppointmentStatus {
  IN_PROGRESS = 'in_progress',
  COMPLETED = 'completed',
  CANCELLED = 'cancelled'
}

@Entity('appointments')
export class Appointment {

  @PrimaryGeneratedColumn()
  id: number;

  @OneToOne(type => Reservation, reservation => reservation.appointment, { cascadeUpdate: true, nullable: false })
  @JoinColumn({ name: 'reservation_id' })
  reservation: Reservation;

  @Column('enum', {
    default: AppointmentStatus.IN_PROGRESS,
    enum: [AppointmentStatus.CANCELLED, AppointmentStatus.COMPLETED, AppointmentStatus.IN_PROGRESS]
  })
  status: AppointmentStatus;

  @ManyToOne(type => User, { nullable: false })
  @JoinColumn({ name: 'status_by_user_id' })
  statusByUser: User;

  @Column(type => AppointmentReview, { prefix: 'customer_review' })
  customerReview: AppointmentReview;

  @Column(type => AppointmentReview, { prefix: 'photographer_review' })
  photographerReview: AppointmentReview;

  @CreateDateColumn({ name: 'created_at' })
  createdAt: Date;

  @UpdateDateColumn({ name: 'updated_at' })
  updatedAt: Date;
}
```

When I do schema sync it produces `photographerReview_rate` instead of `photographer_review_rate`

Do I have a wrong implementation here?

**Comment by pleerock** [comment]
Why it shell produce photographer_review_rate? Typeorm uses camelCase column names thats why `photographerReview` is correct naming and `_rate` is with lodash only because you added lodash in rate naming

**Comment by tontonskie** [comment]
So does it mean `prefix` doesn't change its column name ?

I thought if prefix is supplied it will use it instead of the property name

**Comment by pleerock** [comment]
Sounds like a bug. Im on vacation now, will take a look closely next month. 

BTW instead of 

```ts
  @Column('enum', {
    default: AppointmentStatus.IN_PROGRESS,
    enum: [AppointmentStatus.CANCELLED, AppointmentStatus.COMPLETED, AppointmentStatus.IN_PROGRESS]
  })
```

you can simply do:

```ts 
  @Column('enum', {
    default: AppointmentStatus.IN_PROGRESS,
    enum: AppointmentStatus
  })oo
```

**Comment by pleerock** [comment]
Fix for this bug will be released in next 0.1.0-alpha.38 version
